src/lib/serviceRoutines/postSubscribeContext.cpp

```diff
@@ -36,6 +36,15 @@
 /* ****************************************************************************
 *
 * postSubscribeContext - 
+*
+* POST /v1/subscribeContext
+* POST /ngsi10/subscribeContext
+*
+* Payload In:  SubscribeContextRequest
+* Payload Out: SubscribeContextResponse
+*
+* URI parameters
+*   - notifyFormat=XXX    (handled by mongoBackend)
```

Comment:
We dedided to ignore entity::type... I guess that exist and !exist are also ignored, aren't they?